src/main/java/appeng/client/render/cablebus/CableBusBakedModel.java

```diff
@@ -52,6 +53,8 @@
 public class CableBusBakedModel implements IBakedModel
 {
 
+	private final static Map<CableBusRenderState, List<BakedQuad>> CABLE_MODEL_CACHE = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)